docs/source/release/v6.1.0/diffraction.rst

```diff
@@ -50,6 +50,7 @@ Improvements
 Bugfixes
 ########
 
+- Fix the issue with peak profile functions :ref:`Bk2BkExpConvPV <func-Bk2BkExpConvPV>`.
```

Comment:
Could the release notes be a bit more specific e.g. 
Correctly calculate exponential integral in :ref:`Bk2BkExpConvPV <func-Bk2BkExpConvPV>`